web-console/src/views/task-view/tasks-view.tsx

```diff
@@ -528,6 +544,15 @@ ORDER BY "rank" DESC, "created_time" DESC`;
               ? 'No supervisors'
               : supervisorsError || ''
           }
+          filtered={supervisorFilter}
+          onFilteredChange={filtered => {
+            const datasources = filtered.filter(filter => filter.id === 'datasource')[0];
```

Comment:
small nit: `x.filter(...)[0]` is a less good form of `x.find(...)` also the variable name should be singular (like `datasourceFilter`)